src/test/java/com/sap/oss/phosphor/fosstars/data/artifact/ReleaseInfoFromMavenTest.java

```diff
@@ -47,13 +52,17 @@ public void testIfMavenArtifactExist() throws IOException {
 
       provider.update(MAVEN_ARTIFACT, values);
 
-      assertEquals(1, values.size());
+      assertEquals(2, values.size());
       assertTrue(values.has(RELEASED_ARTIFACT_VERSIONS));
       assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).isPresent());
       assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().isUnknown());
       assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().empty());
-      assertEquals(15, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
+      assertEquals(20, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
       assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().get("1.10.10").isPresent());
+      assertFalse(values.of(ARTIFACT_VERSION).get().isUnknown());
+      ArtifactVersion foundArtifactVersion = values.of(ARTIFACT_VERSION).get().get();
+      assertEquals("1.10.10", foundArtifactVersion.getVersion());
+      assertEquals(asLocalDateTime(1618200022000L), foundArtifactVersion.getReleaseDate());
```

Comment:
```suggestion
      assertTrue(values.has(ARTIFACT_VERSION)); // Just a suggestion, you can ignore if you want.
      assertFalse(values.of(ARTIFACT_VERSION).get().isUnknown());
      ArtifactVersion foundArtifactVersion = values.of(ARTIFACT_VERSION).get().get();
      assertEquals("1.10.10", foundArtifactVersion.getVersion());
      assertEquals(asLocalDateTime(1618200022000L), foundArtifactVersion.getReleaseDate());
```

Reply:
If wanted I can add this line.
IMHO I prefer to not add such tests if it is implicit tested by another test (here the `assertFalse(values.of(ARTIFACT_VERSION).get().isUnknown());` would fail in case of no `ARTIFACT_VERSION` is in the values list).
I understand that having more explicit tests helps sometimes to find the reason for a failure quicker. 
But in above sample I do not think that it would help 🙂
However, as mentioned, if wanted I can add this line.